packages/react-instantsearch/src/connectors/connectRefinementList.js

```diff
@@ -100,9 +100,23 @@ export default createConnector({
         isRefined: v.isRefined,
       }));
 
+    let facetValues;
+    if (facetValuesResults) {
+      facetValues = facetValuesResults[attributeName]
+        .slice(0, limit).map(
+          v => ({
+            label: v.value,
+            value: getValue(v.value, props, searchState),
+            _highlightResult: {label: {value: v.highlighted}},
```

Comment:
Tricky one! I feel it would be better to have one that is distinct. But as you did it works and it is compatible with the current hits so it make more intuitive to use when building a custom version of this widget. Keep it like that it's great!